app/controllers/rewards_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class RewardsController < ApplicationController
+  # GET /rewards
+  def index
+    @rewards = Reward.all
+  end
+
+  # GET /rewards/1
+  def show
+    set_reward
```

Comment:
In general in RoR we do not use names like `set_foo`, `get_foo` - simply `foo`
but on the other hand, if we have a helper method, which is used only in one place I would consider not using it at all :) 
```suggestion
    @reward = Reward.find(params[:id])
```

Reply:
I wanted to keep the naming from previous controllers like set_kudo or set_company_value. To me it's a bit more readable for now. 

As for using a method in only one place... here I also wanted to keep the repeatability with the previous controllers. 

So the question is to change or leave it for now?